src/pandablocks_ioc/_pvi.py

```diff
@@ -260,7 +262,7 @@ def configure_pvi(screens_dir: Optional[str], clear_bobfiles: bool):
 
         Pvi._clear_bobfiles = clear_bobfiles
 
-    @staticmethod
+        
```

Comment:
This is still a static method
```suggestion
        @staticmethod
```